srcpkgs/socat/template

```diff
@@ -11,10 +11,12 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://www.dest-unreach.org/socat/"
 distfiles="http://www.dest-unreach.org/socat/download/socat-${version}.tar.bz2"
-checksum=6690a9f9990457b505097a272bbf2cbf4cc35576176f76646e3524b0e91c1763
+checksum=d47318104415077635119dfee44bcfb41de3497374a9a001b1aff6e2f0858007
 
 case "$XBPS_TARGET_MACHINE" in
-	*-musl) CFLAGS="-D_LINUX_IF_ETHER_H";;
+	*-musl) CFLAGS="-D_LINUX_IF_ETHER_H"
+		configure_args+=" sc_cv_getprotobynumber_r=2"
```

Comment:
See [ffa09eac](https://repo.or.cz/socat.git/commit/ffa09eac003f00767147db5ae69134966fa0ded8), [967e2b24](https://repo.or.cz/socat.git/commit/967e2b24b5318f26aedc4cafce6e0ef11c187bb1).

Reply:
Could you link the commits in your commit message?

Reply:
Okay. Done.